### DISCUSSION TITLE
Airflow 2.0.0 db init with MySQL errors

### DISCUSSION THREAD

**Discussion by usernametakenforever** [apache/airflow#13695] [Q&A] [answered]
I'm trying to install Airflow 2.0.0 with MySQL and `airflow db init` gives errors. Below is the stack trace for error in SQL syntax

`sqlalchemy.exc.ProgrammingError: (_mysql_exceptions.ProgrammingError) (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'JSON' at line 1")
[SQL: ALTER TABLE rendered_task_instance_fields ADD COLUMN k8s_pod_yaml JSON]
(Background on this error at: http://sqlalche.me/e/13/f405)`

I'm not sure if this is bug in the code or something in the config related to kubernetes which is causing this issue. I haven't changed any k8s config which is generated by default.

**Comment by kaxil** [comment, marked as answer]
What version of MySQL are you using?

**Reply by usernametakenforever** [reply]
Hi @kaxil, thanks for the reply. Below is mysql version I've setup

`mysqld  Ver 5.6.50 for Linux on x86_64 (MySQL Community Server (GPL))`

```
mysql> SELECT @@VERSION;
+-----------+
| @@VERSION |
+-----------+
| 5.6.50    |
+-----------+
1 row in set (0.00 sec)
```

**Reply by kaxil** [reply]
@usernametakenforever MySQL 5.6 is not supported, we only support MySQL >=5.7 and if you can I would recommend using MySQL 8 or Postgres instead

**Reply by kaxil** [reply]
https://github.com/apache/airflow#requirements

**Reply by usernametakenforever** [reply]
Thank you @kaxil, I upgraded and problem is fixed now.
